charts/kured/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.7.0"
 description: A Helm chart for kured
 name: kured
-version: 2.8.0
+version: 2.8.1
```

Comment:
@dhoeld Could you please set `2.9.0` here? This is a change of the default behavior and should be not only a patch release. :wink:

Reply:
@ckotzbauer Thanks for your review! I set the version 2.9.0 now.